nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/dependencies/P2CUDFDependencyResolver.java

```diff
@@ -159,19 +206,19 @@ public DependencyResolution resolve(List<String> pkgInstall,
         return list.toArray(new PackageDependency[list.size()]);
     }
 
+    /**
+     * This a backward-compatibility method used only from tests. There is no more possibility to upgrade a single
+     * package without knowing the target version so when calling this method on a already installed package, it will
+     * return a no-change solution
+     *
+     * @deprecated since 1.4.26 Use {@link #resolve(List, List, List, String)} instead
+     */
```

Comment:
What do you mean by "Remove its usage cases" ? Replace by the good method in all tests ?